tutorials/io/ntuple/ntpl004_dimuon.C

```diff
@@ -41,8 +41,10 @@ constexpr char const *kNTupleFileName = "http://root.cern/files/tutorials/ntpl00
 using namespace ROOT::VecOps;
 
 void ntpl004_dimuon() {
+#ifdef R__USE_IMT
    // Use all available CPU cores
    ROOT::EnableImplicitMT();
+#endif
```

Comment:
I think `ROOT::EnableImplicitMT()` is always available, just a no-op when ROOT is built without IMT. In any case, this change is unrelated to this PR and deprecation warnings!